clusterloader2/pkg/measurement/common/dns/dns_performance-k8s-hostnames.go

```diff
@@ -78,6 +82,9 @@ type dnsPerfK8sHostnamesMeasurement struct {
 	qpsPerClient int
 	// testDurationMinutes is the duration in minutes for DNS client pods to run.
 	testDurationMinutes int
+	// additionalManifestFilePath is the optional location of an extra manifest
```

Comment:
I'm not familiar with this framework but is risky to allow passing unstructured configuration to tests

Reply:
Yeah I understand the concern. My rationale here is a want to install GKE specific CRDs that probably don't need to be hard coded into the perf test framework.

I feel like the unstructured configuration should be ok since this is used for running perf tests and shouldn't be running in anyones production environment anyhow.

Reply:
Design-wise I'd say it's acceptable. Although very flexible, nobody is forced to use it. Users will take the responsibility.

@aojea, do you mean there is a security risk?

Reply:
ah, no security, just in the design of tests I always encourage people to be opinionated, tests are meant to be super stable and solid, so you run them over a black box, and you have clear inputs and you know exactly what output you expect, so the black box is correct because you assert on the outputs. Flexibility also creates more complexity to maintain it ...
If the inputs are flexible the outputs start to be flexible too ... is up to you, just sharing, maybe in this case is different ... just take it into account